lib/whitehall_importer/create_edition.rb

```diff
@@ -145,6 +145,8 @@ def create_edition(status:, edition_number:, current:, create_event: nil, last_e
       create_event ||= history.create_event!
       last_event ||= whitehall_edition["revision_history"].last
 
+      editor_ids = history.editors.map { |editor| user_ids[editor] }
```

Comment:
Some of these might come out as nil. Does this cause any problems for the insert?